.gitignore

```diff
@@ -72,6 +72,7 @@ zulip.kdev4
 *.sublime-workspace
 .vscode/
 *.DS_Store
+.virtualenv
```

Comment:
Can you remove this change from this commit?  If we do need it, it should be in its own commit with a commit message explaining why it is needed.

Reply:
Oops, will remove that